README.md

```diff
@@ -138,19 +170,17 @@ Or you can toggle it inside IRB/Pry:
 >> MethodFinder.toggle_debug!
 ```
 
-## Warning
+## Development
 
-Common sense not included!
+After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake test` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
 
-While I never had any problems with this, it's still better to be safe than
-sorry, so use this with caution and maybe not on production data.
+To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb`, and then run `bundle exec rake release`, which will create a git tag for the version, push git commits and tags, and push the `.gem` file to [rubygems.org](https://rubygems.org).
 
-I initially wrote this for the students of the core Ruby course on
-[RubyLearning](http://rubylearning.org), so Rails is not of interest to me (not
-saying it doesn't work there, just that I test in plain IRB/Pry and not with
-the Rails console.
+## Contributing
 
-## Thanks
+Bug reports and pull requests are welcome on GitHub at https://github.com/citizen428/methodfinder.
+
+### Noteworthy contributors
 
 * [Matthew Lucas](https://github.com/lucas-matt) for [first packaging this as a gem](https://github.com/citizen428/methodfinder/pull/1).
 * [Ryan Bates](https://github.com/ryanb) for [suggesting](https://github.com/citizen428/methodfinder/issues/closed#issue/3)
```

Comment:
Broken links:

* https://github.com/citizen428/methodfinder/issues/closed#issue/3 -> https://github.com/citizen428/methodfinder/issues/3
* https://github.com/citizen428/methodfinder/issues/closed#issue/4 -> https://github.com/citizen428/methodfinder/issues/4

Reply:
Removed the whole section because of #19.